system-x/services/kafka/src/main/java/software/tnb/kafka/resource/openshift/OpenshiftKafka.java

```diff
@@ -266,4 +267,14 @@ private void connectionProperties() {
         props.setProperty("ssl.truststore.location", new File(account().trustStore()).getAbsolutePath());
         props.setProperty("ssl.truststore.type", "PKCS12");
     }
+
+    @Override
+    public String defaultOperatorCatalog() {
+        return "redhat-operators";
+    }
+
+    @Override
+    public String defaultOperatorChannel() {
+        return "stable";
```

Comment:
can you please also create constants for these two?

Reply:
done, thanks